test/forms/DeclarationFormMappingSpec.scala

```diff
@@ -277,4 +277,58 @@ class DeclarationFormMappingSpec extends WordSpec
       }
     }
   }
+
+  //govermentprocedures
+  "GovernmentProcedure Mapping" should {
+
+    "bind" when {
+
+      "valid values are bound" in {
+
+        forAll { govermentProcedure: GovernmentProcedure =>
+          Form(governmentProcedureMapping).fillAndValidate(govermentProcedure).fold(
+            _       => fail("form should not fail"),
+            success => success mustBe govermentProcedure
+          )
+        }
+      }
+    }
+
+    "fail" when {
+
+      "Current Code and Previous Code are missing" in {
+
+        Form(governmentProcedureMapping).bind(Map[String, String]()).fold(
+          error => error must haveErrorMessage("To add procedure codes you must provide Current Code or Previous code"),
```

Comment:
if you prefer you could change this to like you have in other tests
```suggestion
          _ must haveErrorMessage("To add procedure codes you must provide Current Code or Previous code"),
```